demo/dashboard/run.py

```diff
@@ -0,0 +1,72 @@
+import gradio as gr
+import pandas as pd
```

Comment:
please create a requirements.txt file in this dir and include everything that needs to be installed. otherwise this won't work on spaces.

Reply:
Yep will do!